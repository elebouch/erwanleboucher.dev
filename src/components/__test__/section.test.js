import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import Section from "../section"

describe("Section", () => {
  it("renders correctly", () => {
    const { container } = render(<Section title="test" />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
