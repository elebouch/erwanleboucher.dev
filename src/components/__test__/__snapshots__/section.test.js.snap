// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders correctly 1`] = `
.c1 {
  width: 100%;
  height: 3px;
  background-color: var(--dark);
  margin: 10px 0;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  color: var(--dark);
}

.c2 {
  font-family: "Archivo Black",sans-serif;
  font-weight: 400;
  font-size: 16px;
  margin-bottom: 10px;
}

<div
  className="c0"
>
  <div
    className="c1"
  />
  <span
    className="c2"
  >
    test
  </span>
</div>
`;
